developer-docs-site/docs/reference/node_liveness_criteria.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Node Liveness Criteria"
+slug: " node_liveness_criteria"
+---
+
+When you participate in the [Aptos Incentivized Testnet](https://medium.com/aptoslabs/aptos-incentivized-testnet-update-abcfcd94d54c), your validator node must pass liveness checks within 24 hours of being deployed, and at a regular cadence onwards. This is required to ensure that your validator node contributes to the health of the overall network and that you become eligible for incentivized testnet rewards. 
```

Comment:
within 24 hours of the start of the notification process

Reply:
notification process?

Reply:
changing to "notification of acceptance into the incentivized testnet program"